### PR TITLE
Assorted pending 0.12 release updates

### LINKED ISSUE #72: ubuntu instructions say that python 3.7+ is not supported
https://kolibri.readthedocs.io/en/latest/install.html#id3

image

Python 3.7 support was added in https://github.com/learningequality/kolibri-installer-debian/pull/55

----

cc @radinamatic

### PR DESCRIPTION
**Pull request by radinamatic**
- [X] Fixes #72
- [X] Addresses the pending comments from #67 
- [X] Optimized all the new screenshots
- [X] Corrected linting errors

Substitution pattern for hard-coded version number to be addressed in a separate PR.

### CODE REVIEW

File: docs/_static/theme_overrides.css [hunk 1]
```diff
@@ -79,14 +81,12 @@ div.figure {
...
 .star-small {
     padding-bottom: 1px !important;
-    /*background-color: #fff;*/
     border-radius: 50%;
     color: #ffc107;
 }
 
 .help {
     padding-bottom: 1px !important;
-    /*background-color: #fff;*/
```
Comment (indirectlylit) [inline docs/_static/theme_overrides.css:89]: cool, thanks for removing these

File: docs/coach.rst [hunk 2]
```diff
@@ -32,27 +32,31 @@ Quizzes
...
 
 * Progress bar will indicate how many learners have started |started| and completed |completed-small| the quiz.
-* You can click on the progress bar itself to open the complete quiz data in the **Reports** tab.
+* You can click on the progress bar to open the complete quiz data in the **Reports** tab.
 * Click :guilabel:`VIEW ALL` to access the full list of notifications about quizzes (only the 3 most recent are displayed in the block).
 
+.. note::
+  **Quizzes** were known as **Exams** in Kolibri versions prior to 0.12.
+
+
```
Comment (indirectlylit) [inline docs/coach.rst:41]: great

File: docs/conf.py [hunk 3]
```diff
@@ -36,6 +36,7 @@
     'https://groups.google.com/a/learningequality.org/forum/#!forum/dev',
     'http://127.0.0.1:8080',
     'http://127.0.0.1:8080/',
+    'https://www.kiwix.org',
```
Comment (indirectlylit) [inline docs/conf.py:39]: was this check failing?

File: docs/install/tutorials/rpi.rst [hunk 4]
```diff
@@ -375,10 +376,10 @@ Kolibri is under development with regards to optimizing performance on Raspberry
 
 This work will be released in a future package targeting a pre-configured UWSGI and Nginx implementation, which will be possible to add by replacing the package of this tutorial with the new package. We will add a release note and update this tutorial accordingly.
 
-You may also want to install other services such as `Kiwix <https://kiwix.org>`__. If you have followed this tutorial, you can install Kiwix alongside Kolibri by downloading the ``kiwix-serve`` package and adding an Nginx configuration similar to the one we added for Kolibri.
+You may also want to install other services such as `Kiwix <https://www.kiwix.org>`_. If you have followed this tutorial, you can install Kiwix alongside Kolibri by downloading the ``kiwix-serve`` package and adding an Nginx configuration similar to the one we added for Kolibri.
```
Comment (indirectlylit) [inline docs/install/tutorials/rpi.rst:379]: switched from double- to single-underscore here. Intentional?

**Review by indirectlylit — APPROVED** [review]
lots of good changes in here. Merge when you're ready

File: docs/conf.py (hunk 3 above)
Comment (radinamatic) [reply docs/conf.py:39]: Yes, for some reason the `make lint` command was repeatedly stumbling on it... 😒

File: docs/install/tutorials/rpi.rst (hunk 4 above)
Comment (radinamatic) [reply docs/install/tutorials/rpi.rst:379]: Turns out Sphinx does not like duplicate explicit target references (TIL), so I had to turn them in anonymous:
https://stackoverflow.com/questions/5464627/how-to-have-same-text-in-two-links-with-restructured-text/14067756#14067756

**Pull request merged**